src/loaders/compute_tools/checkm2/versions.yaml

```diff
@@ -19,4 +19,11 @@ versions:
     date: 2023-11-08
     notes: |
       - create fatal error file for CheckM2
+    reference_db_version: 1.0.1
+  - version: 0.1.4
+    date: 2024-04-05
+    notes: |
+      - Remove the 'threads' and 'program_threads' parameters
+      - and introduce the capability to specify the number of threads per tool run
+      - pass job_id to the tool container and remove node_id
```

Comment:
I don't think you want the hyphen on 2nd line do you? I think that starts a new string

Reply:
👍

Reply:
Not sure if indentation needs to be added or not

Reply:
👍 sure.